crates/p2p/src/client/peer_agnostic/fixtures.rs

```diff
@@ -275,10 +283,12 @@ pub fn class_resp(tag: i32) -> ClassesResponse {
             ClassDefinition::Sierra(s) => Class::Cairo1 {
                 class: s.to_dto(),
                 domain: 0,
+                class_hash: Hash(Felt::default()),
```

Comment:
I guess more randomization is also ok here.

```suggestion
                class_hash: Faker.fake(),
```